lib/sycamore/sycamore/transforms/extract_graph.py

```diff
@@ -46,6 +68,16 @@ def resolve(self, docset: "DocSet") -> "DocSet":
         from sycamore.reader import DocSetReader
         from ray.data.aggregate import AggregateFn
 
+        reader = DocSetReader(docset.context)
+
+        # Get list[Document] representation of docset, trigger execute with take_all()
+        execution = Execution(docset.context, docset.plan)
+        dataset = execution.execute(docset.plan)
+        docs = dataset.take_all(None)
```

Comment:
you can remove the None, a little confusing with it there

Reply:
fixed